src/remote.c

```diff
@@ -309,7 +305,7 @@ static void remote_packet_process_general(char *packet, const size_t packet_len)
 #if ENABLE_DEBUG == 1 && defined(PLATFORM_HAS_DEBUG)
 		debug_bmp = true;
 #endif
-		remote_respond_string(REMOTE_RESP_OK, PLATFORM_IDENT "" FIRMWARE_VERSION);
+		remote_respond_string(REMOTE_RESP_OK, BOARD_IDENT);
```

Comment:
This looks good for the non-dynamic form, but likely needs a `snprintf()` here to include `platform_ident()` in the dynamic case. If we wind up using format strings more often in future remote protocol code, we can add somthing like a `remote_respond_format_string()` or so.. but for now for this, formatting into the packet buffer and using that is fine as in:
```c
#ifndef PLATFORM_IDENT_DYNAMIC
		remote_respond_string(REMOTE_RESP_OK, BOARD_IDENT);
#else
		snprintf(packet, GDB_PACKET_BUFFER_SIZE, BOARD_IDENT, platform_ident());
		remote_respond_string(REMOTE_RESP_OK, packet);
#endif
```

Reply:
Thanks, done.